### DISCUSSION TITLE
Wo und wie kann man erkennen wer gerade in jarvis angemeldet ist?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2591] [General]
Wie kann man mit javascript in scripts abfragen, welche Person gerade aktiv das Frontend nutzt bzw. angemeldet ist?

**Comment by Zefau** [comment]
Du meinst die Benutzer-ID, wenn in jarvis der Login aktiviert ist?

**Comment by mcuiobroker** [comment]
Ja, wenn Benutzer-Login aktiviert ist.
Benutzername und Benutzergruppe (ioBroker) um daraus evtl Berechtigungen festlegen zu können.
Thema TAB sperren, bzw. gar nicht erst anzeigen, für bestimmte Nutzer.

Es scheint keine ID, Username oder so zu geben, welche man im navigator abfragen könnte.
-> Bitte interne Variable `loggedUser` anlegen.
